src/LibraryManager.Contracts/CancellationHelpers.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Web.LibraryManager
```

Comment:
Placing Helper classes in a `Contracts` assembly feels really weird. 
We should ideally be creating a `Common` assembly for this stuff (which would be a good place to put default implementations as well).
Other alternative is to share this code between projects for now.

Reply:
Yes, I agree. I did not want to create a new assembly for that since we are not exposing extensibility for now. This will be part of the extensibility work and we can decide it we want to expose these helper classed or not in the feature. For this specific one, I am more inclined of not exposing at all

Reply:
Yes, I will move these